BedeSlots/BedeSlots/Areas/Identity/Models/AccountViewModels/Validation/MinimumAgeAttribute.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BedeSlots.Areas.Identity.Models.AccountViewModels.Validation
+{
+    public class MinimumAgeAttribute : ValidationAttribute
+    {
+        private readonly int minimumAge;
+
+        public MinimumAgeAttribute(int minimumAge)
```

Comment:
When you inicialize the attribute [smth(someNumber)] uf that someNumbers goes in the constructor then what is passed in IsValid?